akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -83,6 +83,7 @@ object TestPublisher {
 
     type Self <: ManualProbe[I]
 
+    @ccompatUsedUntil213
```

Comment:
weird place for this annotation? but it doesn't matter where it goes so :+1: :smile: